swift_basics/array.playground/Contents.swift

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
In this Playground, you tried with an array initialization at the time of creation. try the example on how to add elements dynamically inside an array.